profile/README.md

```diff
@@ -3,9 +3,9 @@ Welcome to Civo on GitHub</h3>
 <p align="center">
   <img src="https://gpvc.arturio.dev/civo" alt="profile views"> •  
   <a href="https://twitter.com/intent/follow?screen_name=CivoCloud&tw_p=followbutton"><img src="https://img.shields.io/twitter/follow/CivoCloud?label=%40CivoCloud&style=social"></a>  •
-  <a href="https://civo.com">Website</a> •
+  <a href="https://wwww.civo.com">Website</a> •
```

Comment:
This should be `www.`

Reply:
@civo-claire  just checking if you want to push the above change to merge

Reply:
Yes please @kaihoffman